addons/bestja_detailed_reports/models.py

```diff
@@ -171,6 +198,28 @@ def _compute_project_dates(self):
         return "{} — {}".format(date_start, date_stop)
 
     @api.one
+    @api.depends('project', 'report_entries')
+    def _compute_total_cities_nr(self):
+        """
+            Computes number of unique cities where stores
+            are held, both for the organization and all its children.
+        """
+        project = self.sudo().project
+        level = project.organization.level
+        if level == 1:
+            projects_under = project.children
```

Comment:
`projects_under` is only one character shorter than `project.children`. I don't feel there is a need too invent new terminology here (children projects vs. "under projects").